client/src/components/marketLocator/map.tsx

```diff
@@ -1,130 +1,136 @@
-import React, { useState } from "react";
+import TextField from '@material-ui/core/Input';
+import { Button } from "@material-ui/core";
```

Comment:
Same thing, for optimal build time- when we destructure like this, we're pulling in the entire Material package which slows the build down
```suggestion
import Button from "@mui/material/Button";
```